scripts/tools/generate_esp32_chip_factory_bin.py

```diff
@@ -430,19 +443,19 @@ def any_base_int(s): return int(s, 0)
     parser = argparse.ArgumentParser(description='Chip Factory NVS binary generator tool')
 
     # These will be used by CommissionalbeDataProvider
-    parser.add_argument('-p', '--passcode', type=any_base_int, required=True,
+    parser.add_argument('-p', '--passcode', type=any_base_int, required=False,
```

Comment:
required is false by default. Delete it instead of setting to `=False`

Reply:
Deleted all occurrences of `required=False`